GitUI/CommandsDialogs/FormDiff.Designer.cs

```diff
@@ -418,11 +418,9 @@ private void InitializeComponent()
             // 
             // findInDiffToolStripMenuItem
             // 
-            findInDiffToolStripMenuItem.Image = Properties.Images.Preview;
```

Comment:
The icon could be kept:

![image](https://github.com/gitextensions/gitextensions/assets/36601201/dca78dfc-167d-4868-876f-a3f26400b5a5)

Reply:
This should be changed too, for consistency - will push an update to discuss